faith_data/src/commonMain/kotlin/net/thechance/mena/faith/data/repository/location/MobileGeocoderWrapper.kt

```diff
@@ -0,0 +1,15 @@
+package net.thechance.mena.faith.data.repository.location
+
+import dev.jordond.compass.Coordinates
+import dev.jordond.compass.Place
+import dev.jordond.compass.geocoder.MobileGeocoder
+import dev.jordond.compass.geocoder.placeOrNull
+import net.thechance.mena.faith.domain.entity.Mosque
+
+class MobileGeocoderWrapper : GeocoderWrapper {
```

Comment:
```suggestion
class GeocoderImplementation: Geocoder {
```


i didnt get why we use 'wrapper' naming , could u help me to get it